\begin{center}
Gabe Morris
\end{center}

In [1]:
# Notebook Preamble
import sympy as sp
import numpy as np
import matplotlib.pyplot as plt
from IPython.display import display, Markdown

plt.style.use('maroon_ipynb.mplstyle')

\pagebreak
\tableofcontents
\pagebreak

\begin{center}
\begin{tabular}{c c c}
ME 4403 & Homework 2 & Gabe Morris \\
& & gnm54
\end{tabular}
\end{center}

# Problem 3-39
## Given

\begin{center}
\includegraphics{images/fig1}
\end{center}

## Find
For the beam above, find the maximum tensile stress due to $M$ and the maximum shear stress due to $V$.

## Solution
The free body diagram is,

\begin{center}
\includegraphics{images/fig2}
\end{center}

In [2]:
# Getting reaction forces
Ay, By = sp.symbols('A_y B_y')
eq1 = sp.Eq(Ay + By, 1800)
eq2 = sp.Eq(200*Ay, 150*1800)

sol = sp.solve([eq1, eq2], dict=True)[0]
[display(eq) for eq in [eq1, eq2]]
display(Markdown('---'))

for key, value in sol.items():
    display(sp.Eq(key, value))

Eq(A_y + B_y, 1800)

Eq(200*A_y, 270000)

---

Eq(A_y, 1350)

Eq(B_y, 450)

The shear and moment diagram is,

\begin{center}
\includegraphics{images/fig3}
\end{center}

The maximum shear and tensile stress occur at $x=100\ mm$.

In [3]:
# Calculating stress due to bending
M, c = 30_000, sp.S(0.025)
(M*c/(sp.pi/4*c**4)).n()  # in Pa

2444619925.89151

In [4]:
# Calculating the maximum shear stress
V = 750_000
(sp.Rational(4, 3)*V/(sp.pi*c**2)).n()  # in Pa

509295817.894065

\pagebreak

\begin{center}
\begin{tabular}{ccc}
ME 4403 & Homework 2 & Gabe Morris \\
& & gnm54
\end{tabular}
\end{center}

# Problem 3-62 Part A
## Given

\begin{center}
\includegraphics{images/fig4}
\end{center}

The tube is $36\ in$ long and $r_i=r_m=0$. The thickness $t$ is $\frac{1}{16}"$.

## Find
The maximum torque that can be applied and the corresponding angle of twist of the tube.

## Solution
For thin-walled tubes,

$$\tau=\frac{T}{2A_mt}$$
$$\theta_1=\frac{TL_m}{4GA_m^2t}$$

See p. 129 for additional details of the above formulas.

In [5]:
# Calculating the maximum torque
tau_max = 12_000  # lbf in
t = sp.Rational(1, 16)  # thickness in inches
Am = (1 - t)**2
T = tau_max*2*Am*t
T.n()  # lbf in

1318.35937500000

From table A-5, the modulus of rigidity is $11.5\ Mpsi$.

In [6]:
G = 11.5e6
Lm = (1 - t)*4  # total length
L = 36  # inches
phi_1 = T*Lm/(4*G*Am**2*t)*L
(phi_1*180/sp.pi).n()  # in degrees

4.59163394776145

The expression gets multiplied by $L$ because $\theta_1$ is the angle of twist per unit length.

\pagebreak

\begin{center}
\begin{tabular}{ccc}
ME 4403 & Homework 2 & Gabe Morris \\
& & gnm54
\end{tabular}
\end{center}

# Problem 3-64
## Given

\begin{center}
\includegraphics{images/fig5}
\end{center}

$c_1=2\ mm$, $L_1=20\ mm$, $c_2=3\ mm$, $L_2=30\ mm$, $c_3=0\ mm$, and $L_3=0\ mm$. The material is steel and the maximum shear is $\tau_{allow}=12\ ksi$. The angle of twist is the same for each section.

## Find
a. Determine the torque transmitted by each leg and the torque transmitted by the entire section.
b. Determine the angle of twist per unit length.

## Solution
The relationship for open looped geometry is,

$$T_i=\frac{\theta_iGL_ic_i^3}{3}$$
$$\tau_{max}=G\theta_ic_{max}$$

From Table A-5, $G_{steel}=79.3\ GPa$. I will find Part B first because it is required to answer Part A.

### Part B

In [7]:
tau_max = sp.S(82.7371e6)  # shear stress in Pa
G = sp.S(79.3e9)  # modulus of rigidity in Pa
c = [sp.S(c_) for c_ in (0.002, 0.003, 0)]  # in m
L = [sp.S(L_) for L_ in (0.02, 0.03, 0)]  # in m
c_max = max(c)

phi_i = tau_max/(G*c_max)
phi_i  # in rad per m

0.347781000420345

### Part A

In [8]:
T = []
for i in range(len(c)):
    T_i = phi_i*G*L[i]*c[i]**3/3
    display(sp.Eq(sp.Symbol(f'T_{i + 1}'), T_i))
    T.append(T_i)
T = sum(T)
display(sp.Eq(sp.Symbol('T'), T))  # torques in N m

Eq(T_1, 1.47088177777778)

Eq(T_2, 7.446339)

Eq(T_3, 0)

Eq(T, 8.91722077777778)

\pagebreak

\begin{center}
\begin{tabular}{ccc}
ME 4403 & Homework 2 & Gabe Morris \\
& & gnm54
\end{tabular}
\end{center}

# Problem 3-81
## Given

\begin{center}
\includegraphics{images/fig6}
\end{center}

A countershaft carrying two V-belt pulleys is shown in the figure. Pulley $A$ receives power from a motor through a belt with the belt tensions shown. The power is transmitted through the shaft and delivered to the belt on pulley $B$. Assume the belt tension on the loose side at $B$ is 15 percent of the tension on the tight side.

## Find
a. Determine the tensions in the belt on pulley B, assuming the shaft is running at a constant speed.
b. Find the magnitudes of the bearing reaction forces, assuming the bearings act as simple supports.
c. Draw shear-force and bending-moment diagrams for the shaft. If needed, make one set for the horizontal plane and another set for the vertical plane.
d. At the point of maximum bending moment, determine the bending stress and the torsional shear stress.
e. At the point of maximum bending moment, determine the principal stresses and the maximum shear stress.

## Solution
### Part A
The directions of the torques about $A$ and $B$ are,

\begin{center}
\includegraphics{images/fig7}
\end{center}

Since the shaft has no angular acceleration, $T_A=T_B$ (with directions shown above). It should also be noted that $T_1$ must be greater than $T_2$ because the torque shows that the pulley is more tensile at the bottom.

In [9]:
# Solving for T1 and T2
T1, T2 = sp.symbols('T_1 T_2')
T_A = 4*(sp.S(300) - 50)
eq1 = sp.Eq(3*(T1 - T2), T_A)
eq2 = sp.Eq(T2, sp.S(0.15)*T1)

[display(eq) for eq in [eq1, eq2, Markdown('---')]]

sol = sp.solve([eq1, eq2], dict=True)[0]
_ = [display(sp.Eq(key, value)) for key, value in sol.items()]

Eq(3*T_1 - 3*T_2, 1000)

Eq(T_2, 0.15*T_1)

---

Eq(T_1, 392.156862745098)

Eq(T_2, 58.8235294117647)

### Part B

In [10]:
# Solving for the reactions
Oy, Oz, Cy, Cz = sp.symbols('O_y O_z C_y C_z')

eq1 = sp.Eq(300 + 50 - Oy - Cy, 0)  # Forces in y direction
eq2 = sp.Eq(sol[T1] + sol[T2] - Oz - Cz, 0)  # Forces in z direction
eq3 = sp.Eq(8*sp.S(350) - Cy*22, 0)  # Moments about z-axis
eq4 = sp.Eq(-16*(sol[T1] + sol[T2]) + Cz*22, 0)  # Moments about the y-axis

sol2 = sp.solve([eq1, eq2, eq3, eq4], dict=True)[0]
[display(eq) for eq in [eq1, eq2, eq3, eq4, Markdown('---')]]
_ = [display(sp.Eq(key, value)) for key, value in sol2.items()]

Eq(-C_y - O_y + 350, 0)

Eq(-C_z - O_z + 450.980392156863, 0)

Eq(2800 - 22*C_y, 0)

Eq(22*C_z - 7215.6862745098, 0)

---

Eq(C_y, 127.272727272727)

Eq(O_y, 222.727272727273)

Eq(C_z, 327.985739750445)

Eq(O_z, 122.994652406418)

### Part C
The shear and moment diagram for the two planes is,

\begin{center}
\includegraphics{images/fig8}
\end{center}

### Part D

In [11]:
# Getting max bending moment
# At A,
M_A = sp.sqrt(1781.818181818181818**2 + 983.9572195**2)
M_B = sp.sqrt(763.6363636363**2 + 1967.914439**2)
sp.Matrix([M_A, M_B])

Matrix([
[2035.44782366535],
[2110.88316471859]])

The maximum bending moment occurs at point B with a value of $2110.88316471859\ lbf\,in$.

In [12]:
# Getting the bending stress
c = sp.S(0.5)
sig_x = (M_B*c/(sp.pi/4*c**4)).n()
sig_x  # psi

21501.2793570833

In [13]:
# Getting the torsional stress
t_xz = (1000*c/(sp.pi/2*c**4)).n()
t_xz  # in psi

5092.95817894065

### Part E

In [14]:
def mohr(s_x, s_y, t_xy):
    sig_x_, sig_y_, tau_xy_ = sp.symbols(r'\sigma_x \sigma_y \tau_{xy}')
    sig1_, sig2_, tau1, tau2 = sp.symbols(r'\sigma_1 \sigma_2 \tau_1 \tau_2')
    C_, R = sp.symbols('C R')
    phi_p = sp.Symbol(r'\phi_p')

    display(Markdown('**Center and Radius:**'))
    C_value = (s_x + s_y)/2
    C_sym = sp.Eq(C_, (sig_x_ + sig_y_)/2)
    C_eq = sp.Eq(C_sym, C_value, evaluate=False)
    display(C_eq)
    R_value = sp.sqrt(((s_x - s_y)/2)**2 + t_xy**2).n()
    R_sym = sp.Eq(R, sp.sqrt(((sig_x_ - sig_y_)/2)**2 + tau_xy_**2))
    display(sp.Eq(R_sym, R_value, evaluate=False))

    display(Markdown('---'))

    display(Markdown('**Principle Stresses:**'))
    sig1_value = C_value + R_value
    sig2_value = C_value - R_value
    sig1_sym, sig2_sym = sp.Eq(sig1_, C_ + R), sp.Eq(sig2_, C_ - R)
    tau1_sym, tau2_sym = sp.Eq(tau1, R), sp.Eq(tau2, -R)
    display(sp.Eq(sig1_sym, sig1_value, evaluate=False))
    display(sp.Eq(sig2_sym, sig2_value, evaluate=False))
    display(sp.Eq(tau1_sym, R_value, evaluate=False))
    display(sp.Eq(tau2_sym, -R_value, evaluate=False))

    display(Markdown('---'))

    display(Markdown('**Angle of Occurrence:**'))
    c_num = s_x - s_y + 2*t_xy*sp.I
    phi_value = (sp.arg(c_num)*180/sp.pi).n()
    phi_sym = sp.Eq(2*phi_p, sp.atan(2*tau_xy_/(sig_x_ - sig_y_)))
    display(sp.Eq(phi_sym, phi_value, evaluate=False))

mohr(sig_x, 0, t_xz)

**Center and Radius:**

Eq(Eq(C, \sigma_x/2 + \sigma_y/2), 10750.6396785417)

Eq(Eq(R, sqrt(\tau_{xy}**2 + (\sigma_x/2 - \sigma_y/2)**2)), 11895.9857309209)

---

**Principle Stresses:**

Eq(Eq(\sigma_1, C + R), 22646.6254094625)

Eq(Eq(\sigma_2, C - R), -1145.3460523792)

Eq(Eq(\tau_1, R), 11895.9857309209)

Eq(Eq(\tau_2, -R), -11895.9857309209)

---

**Angle of Occurrence:**

Eq(Eq(2*\phi_p, atan(2*\tau_{xy}/(\sigma_x - \sigma_y))), 25.348569568567)